src/amo/components/Link.js

```diff
@@ -18,7 +19,12 @@ export class LinkBase extends React.Component {
   }
 
   render() {
-    const { base, children, to } = this.props;
+    const { base, children, href, to } = this.props;
+
+    if (typeof href === 'string' && href.startsWith('/')) {
+      const linkHref = path.join(base, href);
+      return <Link {...this.props} href={linkHref}>{children}</Link>;
```

Comment:
My concern with `to` is that it will result in a 404 because those URLs don't exist in the front-end yet but it will try to load that route and 404. So I need this rendered as an `<a>` tag that doesn't use `react-router`.

I can add that as a comment and file an issue to remove it.

Alternatively I can create a mapStateToProps for the component relying on this and just use the `api` state, ditching the extra stuff for the Link component. Which do you reckon is best?